build-system/global-configs/canary-config.json

```diff
@@ -5,6 +5,6 @@
   "amp-live-list": 1,
   "amp-ios-overflow-x": 1,
   "amp-experiment": 1,
-  "expAdsenseA4A": 0.05,
-  "expDoubleclickA4A": 0.05
+  "expAdsenseA4A": 1.0,
```

Comment:
yep will require a new PR, my bad. i only noticed after the fact. we don't have a preference either just cuts out a few characters really